qcodes/tests/dataset/test_database_creation_and_upgrading.py

```diff
@@ -151,6 +155,76 @@ def test_perform_actual_upgrade_1_to_2():
         assert len(c.fetchall()) == 2
 
 
+def test_perform_actual_upgrade_2_to_3():
+
+    v2fixpath = os.path.join(fixturepath, 'db_files', 'version2')
+
+    if not os.path.exists(v2fixpath):
+        pytest.skip("No db-file fixtures found. You can generate test db-files"
```

Comment:
could you remind me why we skip instead of fail? i'm asking because we have around 40 tests as skipped, and it seems that nobody looks at it. I wouldn't want to miss it when this test lands in that bucket of skipped tests.

Reply:
i see. so you're ok with the fact that the users will test all of the qcodes, except the upgraders? i don't mind.

well, we have quite many tests that are skipped, hence i don't even bother looking up what has been skipped and what is not, hence if the upgrader test will get accidentally skipped, i wouldn't notice, but a possibly broken test might slip through.

Reply:
I think we should consider the sizes of the db's used. If they are "small" we can simply check in the fixtures after they are generated and avoid running this scrip every time